nautobot_circuit_maintenance/urls.py

```diff
@@ -100,20 +100,12 @@
         name="parsednotification",
     ),
     # Notification Source
-    path("settings/", views.NotificationSourceListView.as_view(), name="notificationsource_list"),
-    path("settings/add/", views.NotificationSourceEditView.as_view(), name="notificationsource_add"),
-    path("settings/<uuid:pk>/edit/", views.NotificationSourceEditView.as_view(), name="notificationsource_edit"),
-    path("settings/<uuid:pk>/delete/", views.NotificationSourceDeleteView.as_view(), name="notificationsource_delete"),
+    path("source/", views.NotificationSourceListView.as_view(), name="notificationsource_list"),
+    path("source/<uuid:pk>/edit/", views.NotificationSourceEditView.as_view(), name="notificationsource_edit"),
+    path("source/<uuid:pk>/", views.NotificationSourceView.as_view(), name="notificationsource"),
```

Comment:
Now that we have a `slug` for `NotificationSource`, could use it rather than the `pk` in these URL patterns.